smartcontractkit/external-adapters-js
# BLuna Feed

## Closes 22678

**This PR is dependent on the [BEth PR](https://github.com/smartcontractkit/external-adapters-js/pull/1262).  Only merge this after that PR has been merged in**

## Description

Implements the adapter to support the BLuna price feed

## Changes

- Add handler for `BLuna` to the price adapter

## Steps to Test

1. Run price adapter
2. Send request to adapter and verify that it returns a 200
3. Try step 2 with the quote as `BTC` and `ETH`.

Sample request

```
{
  "id": "1",
  "data": {
    "from": "BLuna",
    "to": "USD",
    "source": "coingecko",
    "terraBLunaContractAddress": "terra1mtwph2juhj0rvjz7dy92gvl6xvukaxu8rfv8ts"
  }
}
```

## Quality Assurance
